packages/tools/piw-utils-internal/src/utils/DataGridFilterIcons.ts

```diff
@@ -6,13 +6,13 @@ export const endsWithIcon = `<?xml version="1.0" encoding="UTF-8"?><svg width="3
 
-export const greaterThanIcon = `<?xml version="1.0" encoding="UTF-8"?><svg width="32px" height="32px" viewBox="0 0 32 32" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><title>Greater Than</title><g id="Greater-Than" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"><path d="M10.718125,15.1934411 L20.1371875,18.9393786 C20.711875,19.1278161 21.346875,18.8600036 21.5553125,18.3409411 L21.933125,17.4009411 C22.141875,16.8818786 21.845,16.3081286 21.2703125,16.1196911 L15.7878125,13.9931286 L21.2740625,11.8653161 C21.845625,11.6778161 22.1409375,11.1068786 21.9334375,10.5906286 L21.5575,9.65562863 C21.35,9.13906613 20.718125,8.87250363 20.1465625,9.06000363 L10.718125,12.8087536 C10.2878125,12.9500036 10,13.3278161 10,13.7518786 L10,14.2506286 C10,14.6743786 10.2878125,15.0521911 10.718125,15.1934411 Z M22.25,20.9996911 L9.75,20.9996911 C9.3359375,20.9996911 9,21.3353161 9,21.7496911 L9,23.2496911 C9,23.6637536 9.3359375,23.9996911 9.75,23.9996911 L22.25,23.9996911 C22.6640625,23.9996911 23,23.6637536 23,23.2496911 L23,21.7496911 C23,21.3353161 22.6640625,20.9996911 22.25,20.9996911 Z" id="greater-than" fill="#264AE5" fill-rule="nonzero" transform="translate(16.000000, 16.499846) scale(-1, 1) translate(-16.000000, -16.499846) "></path></g></svg>`;
+export const greaterThanIcon = `<?xml version="1.0" encoding="UTF-8"?><svg width="32px" height="32px" viewBox="0 0 32 32" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><title>Greater</title><g id="Greater" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"><path d="M10.7181297,17.1934819 L20.1372542,21.94 C20.7119455,22.1284387 21.3469497,21.8590659 21.5553886,21.34 L21.9332036,20.4 C22.141955,19.8809341 21.845078,19.3084387 21.2703867,19.12 L14.787851,15.9931615 L21.2741367,12.87 C21.845703,12.6824988 22.1410174,12.1062534 21.9335161,11.59 L21.5575761,10.66 C21.3500747,10.1434341 20.7181956,9.87249877 20.1466293,10.06 L10.7181297,14.8087787 C10.2878144,14.9500296 10,15.3278446 10,15.7519099 L10,16.2506632 C10,16.674416 10.2878144,17.052231 10.7181297,17.1934819 Z" id="greater" fill="#264AE5" transform="translate(16.000000, 16.000324) scale(-1, 1) translate(-16.000000, -16.000324) "></path></g></svg>`;
```

Comment:
Nice to have 🎄 : move this file into an assets folder

Reply:
Done